build/dockerfiles/Dockerfile.rhel

```diff
@@ -41,9 +42,10 @@ RUN chmod a+rwX            /run/httpd
 RUN mkdir /var/www/html/devfiles
 COPY .htaccess README.md /var/www/html/
 COPY --from=builder /build/devfiles /var/www/html/devfiles
-USER 0
-RUN chmod -R g+rwX /var/www/html/devfiles
 
 STOPSIGNAL SIGWINCH
 
-ENTRYPOINT ["/usr/sbin/httpd", "-D", "FOREGROUND"]
+COPY ./build/dockerfiles/rhel.entrypoint.sh ./build/dockerfiles/entrypoint.sh /usr/local/bin/
+RUN chmod g+rwX /usr/local/bin/entrypoint.sh /usr/local/bin/rhel.entrypoint.sh
+ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
+CMD ["/usr/local/bin/rhel.entrypoint.sh"]
```

Comment:
you're still inconsistent with the Dockerfile approach. My thought was to have fewer differences between Dockerfile and rhel.Dockerfile so maintaining them both would be easier. Wasn't just to nest the command for the sake of nesting... apologies if that wasn't clear/obvious.

Reply:
I view this as consistent between the two; we already have to so much additional work in the rhel dockerfile because rhel, and I view `rhel.entrypoint.sh` as another facet of that problem.

Tacking the run-httpd script onto entrypoint.sh is pointless; the entrypoint should work for both images and do the devfile-registry specific setup that needs to happen before starting. The command can do the container specific starting of the Apache server, in a container specific way (since rhel containers demand weirdness).

In my view: 

- dockerfile entrypoint = application specific startup requirements (rewrite meta.yamls)
- dockerfile cmd = start whatever process we need in the container

Reply:
OK works for me.